libensemble/executors/mpi_executor.py

```diff
@@ -1,38 +1,38 @@
 """
-This module launches and controls the running of MPI jobs.
+This module launches and controls the running of MPI tasks.
```

Comment:
This raises a concern over using "tasks". As MPI tasks refers usually to MPI ranks so this could be confusing. I would say we change this to MPI applications.